src/components/common/Cards/latest.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable global-require */
```

Comment:
@espoirMur Why are using disabling this rule from here instead of from .eslint?

Reply:
done